app.py

```diff
@@ -56,6 +63,26 @@ def get_by_id(id_):
     except Exception as e:
 	    return(str(e))
 
+@app.route("/members_by_state/<state_>")
+def get_members_by_state(state_):
+    headers = {'X-API-Key': os.getenv('PROP_API')}
+    URL = f'https://api.propublica.org/congress/v1/members/senate/{state_}/current.json'
+    response = requests.get(URL, headers = headers).json()
+    results = response['results']
+
+    objects = map(lambda result: MemberIndex(first_name = result['first_name'],
+                                             party = result['party'],
+                                             role = result['role'],
+                                             last_name =result['last_name']),
+                                             results)
+    list_of_members = list(objects)
+
+    try:
+        return jsonify({"results": list(map(lambda member: member.serialize(), list_of_members))})
+
+    except Exception as e:
+	    return(str(e))
+
```

Comment:
We need to figure out how to all the senators and house reps together in one result.